recipes/newrelic/apm/ruby/linux.yml

```diff
@@ -0,0 +1,36 @@
+# Visit our schema definition for additional information on this file format
+# https://github.com/newrelic/open-install-library/blob/main/docs/recipe-spec/recipe-spec.md#schema-definition
+
+name: ruby-agent-installer
+displayName: Ruby Agent Installer
+description: New Relic install recipe for instrumenting Ruby applications
+repository: https://github.com/newrelic/newrelic-ruby-agent
+
+installTargets:
+  - type: application
+    os: linux
+
+keywords:
+  - Apm
+  - ruby
+
+processMatch:
+  - ruby
+
+preInstall:
+  requireAtDiscovery: |
+    exit 132

```

Comment:
Just wondering - should there be more of a check here like there is for Python? Seems like we'll always report back `DETECTED` if there's a process with `ruby` in the cmdline

Reply:
Correct, there aren't as many ruby apps running that are not created by a user. 
In python there are a lot of common tools that are there, which we don't want to signal the user they should instrument their "python" app. It's not as strong of a case for ruby.
Here is an example of python process I'm seeing on an EC2/ubuntu, whereas there are no ruby processes
/usr/bin/networkd-dispatcher
/usr/share/unattended-upgrades/unattended-upgrade-shutdown